pkg/pipeline/transform/connection_tracking/connection_tracking.go

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright (C) 2021 IBM, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package connection_tracking
+
+import (
+	"container/list"
+	"fmt"
+	"hash/fnv"
+	"sync"
+	"time"
+)
+
+type cacheKey uint32
+
+type cacheInfo struct {
+	key             cacheKey
+	flowlogsCount   int
```

Comment:
What is the purpose of the `flowlogsCount` field? It doesn't seem to be used at present.

Reply:
correct, the plan going forward is to use that field and others (like flow bytes) to add another extract phase that will be topx  ... so we will be able to report the flows with "top amount of packets" , "top amount of bytes" etc. ...